docs/source/tutorials_source/package/tutorial_moco_memory_bank.py

```diff
@@ -332,14 +332,15 @@ def validation_step(self, batch, batch_idx):
         _, predicted = torch.max(y_hat, 1)
         num = predicted.shape[0]
         correct = (predicted == y).float().sum()
+        self.validation_step_outputs.append((num, correct))
         return num, correct
 
-    def validation_epoch_end(self, outputs):
+    def on_validation_epoch_end(self):
         # calculate and log top1 accuracy
-        if outputs:
+        if self.validation_step_outputs:
```

Comment:
I think you have to call `self.validation_step_outputs.clear()` at the end of this function. Otherwise outputs are aggregated over all epochs.